src/components/elements/select-list.tsx

```diff
@@ -62,10 +62,10 @@ function CustomOption(props: OptionProps) {
       {...otherProps}
       id={id}
       className={twMerge(
-        "mx-5 mb-5 flex cursor-pointer items-center overflow-hidden rounded-full px-5 py-2 hocus:bg-press-bay-light hocus:text-black hocus:underline",
+        "rs-mt-0 rs-m-neg1 flex cursor-pointer items-center overflow-hidden rounded-full border-3 border-transparent px-8 py-4 hocus:bg-press-bay-light hocus:text-black hocus:underline",
```

Comment:
For some reason when I add `text-16` here, the first `li` item is missing the `text-16` and inheriting the default font size of 2.1rem 🤔 

I'll circle back later to test other possible solutions but for now I added `*:text-16` to the `ul`